testcases/OpTestEEH.py

```diff
@@ -82,8 +94,29 @@ def gather_logs(self):
         c.run_command(cmd)
         c.run_command("dmesg")
 
+    def check_eeh_hit(self):
+        c = self.cv_SYSTEM.sys_get_ipmi_console()
+        res = c.run_command("dmesg")
+        res = ' '.join(res)
+        if 'EEH: Frozen' in res:
+            print "HIT"
+            return True
+        else:
+            print "MISS"
+            return False
+
+    def check_eeh_removed(self):
+        c = self.cv_SYSTEM.sys_get_ipmi_console()
+        res = c.run_command("dmesg")
```

Comment:
same as above, do "dmesg|grep 'permanently disabled'" instead, so to save traffic over IPMI